README.md

```diff
@@ -31,13 +31,13 @@ run distributed or non-distributed TensorFlow/PyTorch/Apache MXNet/XGBoost/MPI j
 ### Master Branch
 
 ```bash
-kubectl apply -k "github.com/kubeflow/training-operator/manifests/overlays/standalone"
+kubectl apply --server-side -k "github.com/kubeflow/training-operator/manifests/overlays/standalone"
```

Comment:
To avoid the following apply errors, we need to use SSA.

```shell
The CustomResourceDefinition "pytorchjobs.kubeflow.org" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
```

Reply:
Why did this occur now? This is bit tricky to add this as users might not try it easily.  eg: [ kubeflow manifests ](https://github.com/kubeflow/manifests) do not have such instructions

Reply:
The `.metadata.annotations.last-applied-configuration` size increased since we added a new field, `selector`, in this PR. That means CRD for PyTorchJob exceeds the limitation of `.metadata.annotations` size.

So we can select the following way:

1. Like the above, add the `--server-side` flag to the apply command or use create command, not apply command.
2. We remove all descriptions from CRD (we can remove descriptions by an additional controller-gen option).

@johnugeorge What do you think?

Reply:
If we select option 2, we should merge another PR to remove CRD descriptions to clarify changes in CHANGELOG and commit history before we merge this PR into the master branch.

Reply:
Does it help if we can trim down descriptions for the field instead of removing all descriptions ?

Reply:
> Does it help if we can trim down descriptions for the field instead of removing all descriptions ?

Yes, we can trim down descriptions (e.g., 100 words) by controller-gen.

Reply:
ASAP, I will create another PR for that. Let's optimize the number of words in that PR.

Reply:
https://github.com/kubeflow/training-operator/blob/6383a9193d4664527defa7c1d171e3033f114a9e/pkg/apis/kubeflow.org/v1/pytorch_types.go#L115

We can keep just the first sentence here as it refers to `autoscalingv2.MetricSpec`

Reply:
Makes sense.

Reply:
I tried to trim comments except for the first sentence. Although, I face the same error.

Reply:
#1735

Reply:
Resolved.